exasol/python_extension_common/deployment/extract_validator.py

```diff
@@ -113,8 +113,7 @@ def verify_all_nodes(self, schema: str, language_alias: str, bfs_archive_path: b
         still nodes pending, for which the extraction could not be verified,
         yet.
         """
-        # manifest = f"{bfs_archive_path.as_udf_path()}/{MANIFEST_FILE}"
-        manifest = "/exaudf/exaudfclient_py3"
+        manifest = f"{bfs_archive_path.as_udf_path()}/{MANIFEST_FILE}"
```

Comment:
Careful, activating this feature might break downstream projects such as the AI-Lab, of transformers-extensions hasn't already released a SLC with a manifest and didn't properly pin the dependency to PEC in the releases.

Reply:
We probably should also go a major version change with that. At least, it would be, if we would be already at 1.0.

Reply:
Regarding transformers-extension, for example, a major version change would not make any difference in the first place, as [pyproject.toml](https://github.com/exasol/transformers-extension/blob/main/pyproject.toml#L30) contains
> exasol-python-extension-common = ">=0.3.0"

Reply:
I thought so, this needs to change. No, unbounded >= anymore.

Reply:
So, this PR is blocked until, we released all downstream projects with either a manifest file in the SLC or a dependency limitation